jetty-server/src/main/java/org/eclipse/jetty/server/DebugListener.java

```diff
@@ -185,7 +185,7 @@ protected void log(String format, Object... arg)
         long now = System.currentTimeMillis();
         long ms = now % 1000;
         if (_out != null)
-            _out.printf("%s.%03d:%s%n", __date.formatNow(now), ms, s);
+            _out.printf("%s.%03d:%s%n", __date.format(now), ms, s);
```

Comment:
There is a big semantic difference between `format` and `formatNow`.  The later is expected to be a time near now and thus will update the tick, the former could be anytime, so it will use the tick if it happens to be in the same tick, but it will not update the tick!